public/app/types/dashboard.ts

```diff
@@ -81,3 +81,17 @@ export interface DashboardState {
   modifiedQueries: QueriesToUpdateOnDashboardLoad | null;
   panels: { [id: string]: PanelState };
 }
+
+export enum DashboardSource {
+  Gcom = 0,
+  Json = 1,
+}
+
+export interface ImportDashboardState {
+  meta: { updatedAt: string; orgName: string };
+  dashboard: any;
+  source: DashboardSource;
+  inputs: any[];
+  gcomError: string;
+  isLoaded: boolean;
+}
```

Comment:
it's fine to have these defined in the reducers file I think, makes it easier and less spread out. We don't reuse this type in order places, what do you think? We do the same in a few places already.

Reply:
Yeah, makes sense